app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,70 @@
+# Welsh translations for PROJECT.
+# Copyright (C) 2022 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2022-10-26 13:26+0100\n"
+"PO-Revision-Date: 2022-10-25 16:55+0100\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: cy\n"
+"Language-Team: cy <LL@li.org>\n"
+"Plural-Forms: nplurals=5; plural=(n==1 ? 1 : n==2 ? 2 : n==3 ? 3 : n==6 ?"
+" 4 : 0);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.10.3\n"
+
+#: app/create_app.py:79
+msgid "Apply for funding to save an asset in your community"
+msgstr "Gwneud cais am gyllid i achub ased yn eich cymuned"
+
+#: app/templates/dashboard.html:10
+#, python-format
+msgid " %(count)s application "
+msgid_plural " %(count)s applications "
+msgstr[0] " %(count)s "
+msgstr[1] " %(count)s "
+msgstr[2] " %(count)s "
+msgstr[3] " %(count)s "
+msgstr[4] " %(count)s "
```

Comment:
We should have the translation for applications here, or leave blank.

Reply:
leaving blank for the time being as this is not part of this ticket 👍🏾